src/dsmlp/app/validator.py

```diff
@@ -51,6 +54,24 @@ def handle_request(self, request: Request):
         return self.admission_response(request.uid, True, "Allowed")
 
     def validate_pod(self, request: Request):
+
+        ### if tgpt-validator == enabled
+        ### run special tritongpt validator that gets permitted UIDs from namespace instead of sicad
+        
+        try:
+            namespace = self.kube.get_namespace(request.namespace)
```

Comment:
to streamline logic - how about something like this?
` [#](url)

    def validate_pod(self, request: Request):

        ### if tgpt-validator == enabled
        ### run special tritongpt validator that gets permitted UIDs from namespace instead of sicad

        try:
            namespace = self.kube.get_namespace(request.namespace)
            tgpt_label = self.kube.get_tgpt_label(namespace)
            
            if(tgpt_label == "enabled"):
                self.logger.info("Triton GPT Mode Activated. Only running TritonGPT Validator.")
                TritonGPTValidator(self.kube, self.logger).validate_pod(request)
                return

        except KeyError: as err:
            self.logger.debug("Failed to evaluate TGPT label logic. Falling back on regular validator components. Error: " + str(err)
            pass
            
        for component_validator in self.component_validators:
            component_validator.validate_pod(request)
`